engine-api/src/methods/get_proof.rs

```diff
@@ -0,0 +1,159 @@
+use {
+    crate::{
+        json_utils::{self, access_state_error},
+        jsonrpc::JsonRpcError,
+    },
+    alloy::{
+        eips::{BlockId, BlockNumberOrTag},
+        primitives::{Address, U256},
+    },
+    moved::types::{
+        queries::ProofResponse,
+        state::{Query, StateMessage},
+    },
+    tokio::sync::{mpsc, oneshot},
+};
+
+pub async fn execute(
```

Comment:
The `execute` method is missing in `request.rs` and the naming is missing in `method_name.rs` to redirect the client calls.

Reply:
Good catch, thanks. Fixed in https://github.com/MovedNetwork/op-move/pull/186/commits/e3c1a30c897f53053ae7bff1496c3cfd9b738d6e